packages/@react-spectrum/table/src/table.css

```diff
@@ -49,3 +64,148 @@
+    padding: 0 var(--spectrum-global-dimension-size-100);
+    border-radius: var(--spectrum-alias-border-radius-regular);
+    margin-inline-end: var(--spectrum-global-dimension-size-100);
+    margin-top: auto;
+    margin-bottom: auto;
+    height: fit-content;
+  }
+}
+
+.react-spectrum-Table-body--dropTarget {
+  border-color: var(--spectrum-table-dropindicator-border-color);
+  background-color: var(--spectrum-alias-highlight-selected);
+  box-shadow: inset 0 0 0 1px var(--spectrum-table-cell-border-color-key-focus);
+}
+
+.react-spectrum-Table-InsertionIndicator {
+  z-index: 3;
+  width: calc(100% - (2 * var(--spectrum-table-dropindicator-circle-size)));
+  inset-inline-start: var(--spectrum-table-dropindicator-circle-size);
+  position: absolute;
+  outline: none;
+
+  &.react-spectrum-Table-InsertionIndicator--dropTarget {
+    background: var(--spectrum-table-dropindicator-border-color);
+    border-bottom: 2px solid var(--spectrum-table-dropindicator-border-color);
+
+    /* forced-color-adjust: none, so that background color
+    will render border for insertion indicator. */
+    forced-color-adjust: none;
+
+    &:before {
+      left: calc(var(--spectrum-table-dropindicator-circle-size) * -1);
+    }
+
+    &:after {
+      right: calc(var(--spectrum-table-dropindicator-circle-size) * -1);
+    }
+
+    &:before,
+    &:after {
+      content: '';
+      position: absolute;
+      top: calc(var(--spectrum-table-dropindicator-circle-size) * -1 / 2 + var(--spectrum-table-dropindicator-border-size) / 2);
+      width: var(--spectrum-table-dropindicator-circle-size);
+      height: var(--spectrum-table-dropindicator-circle-size);
+      border-radius: 50%;
+      border: var(--spectrum-table-dropindicator-border-size) solid;
+      box-sizing: border-box;
+      border-color: var(--spectrum-table-dropindicator-circle-border-color);
+      background-color: var(--spectrum-table-background-color);
+      z-index: 5;
+
+      /* forced-color-adjust: none, so that box-shadow and background-color styles will render */
+      forced-color-adjust: none;
+    }
+  }
+}
+
+.react-spectrum-Table-dragButtonCell {
+  padding: var(--spectrum-global-dimension-size-40);
+}
+
+.spectrum-Table-row-draghandle-button {
+  border-radius: var(--spectrum-alias-border-radius-regular);
+  padding: var(--spectrum-global-dimension-size-10);
+  
+  &:focus-ring {
+    box-shadow: inset 0 0 0 2px var(--spectrum-global-color-blue-400);
+    outline: none;
+
+    /* forced-color-adjust: none, so that box-shadow style will render */
+    forced-color-adjust: none;
+  }
+  svg {
+    margin: auto 0;
+    display: block;
+    padding: var(--spectrum-global-dimension-size-40);
+  }
+}
+
+.react-spectrum-dragButtonHeadCell {
+  padding: 0;
+  height: inherit;
+}
+
+.react-spectrum-Table-row--dropTarget {
+  border: 1px var(--spectrum-table-dropindicator-border-color);
+  box-shadow: inset 2px 0 0 0 var(--spectrum-table-dropindicator-border-color),
```

Comment:
what are all these 2 and -3 px values? can we use any variables for them? do they vary from medium to large?

Reply:
These are the same as ListView has. I don't think they are dependent on scale since it is basically a focus ring.

The reasoning behind the one -3px value, is because the bottom divider between rows will otherwise cover up 1px of the box shadow like so:

<img width="57" alt="Screen Shot 2023-03-14 at 4 20 32 PM" src="https://user-images.githubusercontent.com/8961049/225139702-4416209c-549d-412f-ba5a-b2a15343a607.png">

Reply:
ok, if it exists in listview, that's good enough for me. thanks for clearing that up